orttraining/orttraining/python/training/ortmodule/_utils.py

```diff
@@ -50,6 +51,33 @@ def _ortvalue_to_torch_tensor(ortvalue, device):
     dlpack_tensor = ortvalue.to_dlpack()
     return _torch_tensor_from_dl_pack(dlpack_tensor, ortvalue, device)
 
+
+def _ortvalues_to_torch_tensor(ortvalues, device):
+    if len(ortvalues) == 0:
+        return tuple()
+    if (hasattr(ortvalues, 'to_dlpack') and
+            all(ortvalue.data_type() != 'tensor(bool)' for ortvalue in ortvalues)):
```

Comment:
Why do we exclude bool here? I think we can still use the new way to generate the torch tensor tuples and then apply the conversion by checking tensor element type after that.